cli/Cargo.toml

```diff
@@ -79,7 +79,7 @@ node-alias = ["dirs", "reqwest"]
 health = ["reqwest", "serde_json"]
 keygen = ["dirs", "whoami"]
 
-database = ["splinter/database", "diesel", "postgres"]
+database = ["splinter/postgres", "diesel", "postgres"]
```

Comment:
Does the cli still need to have diesel and postgres as direct dependencies?

Reply:
I will investigate this separately.